### ISSUE TITLE
createFromFormat not compatible - PHP 8

### ISSUE THREAD

**Issue by axelf** [fightbulc/moment.php#214]
Hi,

i use PHP 8.1 and get this message:
`
Return type of Moment\Moment::createFromFormat($format, $time, $timezone = null, ?Moment\FormatsInterface $formatsInterface = null) should either be compatible with DateTime::createFromFormat(string $format, string $datetime, ?DateTimeZone $timezone = null)`

**Comment by Moreniell** [comment]
Same for other overridden methods:
```
Deprecated: Return type of Moment\Moment::createFromFormat($format, $time, $timezone = null, ?Moment\FormatsInterface $formatsInterface = null) should either be compatible with DateTime::createFromFormat(string $format, string $datetime, ?DateTimeZone $timezone = null): DateTime|false, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice
Deprecated: Return type of Moment\Moment::format($format = null, $formatsInterface = null) should either be compatible with DateTime::format(string $format): string, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice
Deprecated: Return type of Moment\Moment::setTimezone($timezone) should either be compatible with DateTime::setTimezone(DateTimeZone $timezone): DateTime, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice
Deprecated: Return type of Moment\Moment::setTime($hour, $minute, $second = null, $microseconds = null) should either be compatible with DateTime::setTime(int $hour, int $minute, int $second = 0, int $microsecond = 0): DateTime, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the noticeDeprecated: Return type of Moment\Moment::setDate($year, $month, $day) should either be compatible with DateTime::setDate(int $year, int $month, int $day): DateTime, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice
```

**Comment by localheinz** [comment]
It appears that this was already fixed with #209.
